resources/views/admin/categories/index.blade.php

```diff
@@ -0,0 +1,59 @@
+@extends('shop.shop_layouts.master')
+@section('content')
+<h3>List of Categories</h3>
+<hr>
+<div class="panel panel-default">
+  <!-- Default panel contents -->
+  <div class="panel-heading">Panel heading</div>
+  <div class="panel-body">
+      <div class="col-sm-6">
```

Comment:
@afnanali1992 Where's the div close tag here??
Please check all syntax and use pretty-css to make the code look nicer, now it's hard to read!